client/test/grammars/test-cases/keywords.bb

```diff
@@ -51,7 +158,14 @@
  
 >True
 #^^^^ source.bb constant.language.python.bb
- 
+
+>Trueabc
+#^^^^ - constant.language.python.bb
+
+>abcTrue
```

Comment:
This looks like it should be detected as a variable?

Reply:
this is a negative check to make sure it does not have the scope. 

For the variables, I think it makes sense to only show identifiers as variables when they are followed by a valid operator like `=`, `.=` etc. You will see this behaviour in a lot of major languages (JS, python).